### PR TITLE
Img classification/fix missing artifacts wheel

### PR DESCRIPTION
**Pull request by abhishekpratapa**
- fixes missing artifacts in build wheel.

### CODE REVIEW

**Comment by abhishekpratapa** [comment]
Fixes #1520

File: src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js [hunk 1]
```diff
@@ -29,12 +34,32 @@ class ImageContainer extends Component {
 		}
 	}
 
+	resizeImage(width, height) {
+		if((width/height) > 1){
+			return {"width": "auto", "height": 90, "top": 0, "left": parseInt((-1*(((width/height)*90)-90)/2), 10)};
```
Comment (shantanuchhabra) [inline src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js]: Can you add a comment here explaining what the math is? Also, I would recommend storing `90` in a local variable called `dimension`/`size` if you need it more than once (and maybe even if you need it only once).

File: src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js [hunk 2]
```diff
@@ -4,13 +4,18 @@ import { isRegExp } from 'util';
...
-    super(props);
+  	constructor(props) {
+		super(props);
 	}
 	
-	image_click = () =>{
-		this.props.toggleInfiniteScroll();
-		this.props.updateIncrementalCurrentIndex(this.props.value);
+	image_click = (e) =>{
+		if (e.shiftKey) {
```
Comment (shantanuchhabra) [inline src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js]: Which click is this? Is this on the icon above to toggle infinite scroll?

Comment (abhishekpratapa) [reply src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js]: This click is the one to shift select multiple items. I'll refactor to follow the specifications that finder uses

File: src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js (hunk 1 above)
Comment (abhishekpratapa) [reply src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js]: Yeah! will do!

**Pull request merged**